.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -60,14 +60,8 @@ jobs:
         stage:
```

Comment:
Remove the `stage` parameter as you did for [.github/workflows/build-kata-static-tarball-s390x.yaml](https://github.com/kata-containers/kata-containers/pull/9550/commits/9b8805432b3a911bc91056b542f954a17a2863f3#diff-75d5ecb9b253be81b84649635e2d463572d82f18fe2fee5a51ed96225e237fed) ?

Reply:
We still need it to filter out:
```
          - asset: cloud-hypervisor-glibc
            stage: release
```
which the s390x build doesn't need?

Reply:
ok, missed that.